classes/spec_extensions.rb

```diff
@@ -85,6 +85,8 @@ def or_license_name_and_url
           return { license: "Apache 2", url: "https://www.apache.org/licenses/LICENSE-2.0.html" }
         elsif license.scan(/apache2/).count > 0
           return { license: "Apache 2", url: "https://www.apache.org/licenses/LICENSE-2.0.html" }
+        elsif license.scan(/apache 2/).count > 0
```

Comment:
@segiddins if you go to that link, make sure to turn on the global flag on that regex tester. I guess that setting didn't save, and without it, you won't get to see whether or not it matches all of the lines in the test doc.